lib/validators/qrda_cat3_validator.rb

```diff
@@ -1,4 +1,3 @@
-require 'validators/qrda_cat1_validator'
```

Comment:
Was this require removed because it causes errors, or just because it's unnecessary?